server/utils/pagination.ts

```diff
@@ -0,0 +1,60 @@
+export type Pagination = {
+  previous?: { href: string }
+  next?: { href: string }
+  items?: ({ number: number; href: string; current?: boolean } | { ellipsis: true })[]
+}
+
+/**
+ * Produces parameters for GOV.UK Pagination component macro
+ * NB: `page` starts at 1
+ */
+export function pagination(page: number, pageCount: number, hrefPrefix: string): Pagination {
+  if (pageCount <= 1) {
+    return {}
+  }
+
+  const params: Pagination = {}
+  if (page !== 1) {
+    params.previous = {
+      href: `${hrefPrefix}page=${page - 1}`,
+    }
+  }
+  if (page < pageCount) {
+    params.next = {
+      href: `${hrefPrefix}page=${page + 1}`,
+    }
+  }
+
+  let pages: (number | null)[]
```

Comment:
this bit is to add ellipses to match [GDS’ style](https://design-system.service.gov.uk/components/pagination/#for-larger-numbers-of-pages)